wheel_time_potholes/pothole_reporting/geotag_image.py

```
@@ -0,0 +1,31 @@
+from PIL import Image
+from PIL.ExifTags import TAGS
+
+
+def create_pothole_by_image(image):
+    """
+    create a new pothole using the geotagged data from an image
+    return whether successful creation
+    """
+    gps_data = process_image(image)
+    print(gps_data)
+    # TODO: once formatted, store as new pothole
+
+
+def process_image(image):
+    image = Image.open(image)
+    image.verify()
+
+    image_exif = image._getexif()
```

Might want to add a check to ensure this isn't null before passing it to label_data so an error isn't thrown if the image has no exif data

---

If there's no exif data it should generate a `KeyError` on the line:
`gps_data = label_data(image_exif)['GPSInfo']`
which gets picked up in the views file so it can give a error update on the submission page. I suppose I could change it to check if null and then raise an exception instead of calling `label_data` with null exif data.

---

As long as you're aware that an error occurs it should be fine, I was looking at it with the idea that this would be production use but I realize that's not the case. Should be fine in that case